styles/main/_message.scss

```diff
@@ -208,7 +209,7 @@
 		}
 
 		label input:disabled ~ .label {
-			color: white(0.6);
+			color: white(0.3);
```

Comment:
Is 0.6 too bright? Will this tweak all the other places where we use it as well (e.g., the album visibility settings)?

Reply:
Yes, with 0.6 the change is not sufficiently visible. Not sure what other elements are affected, though. Searching for `attr("disabled"` gives 9 matches (`prop("disabled"` has no match outside this PR except in `node_modules/multiselect-two-sides`, which is impossible to parse).

I will do some more digging but probably best to add a CSS class (which I actually had at one point...)

Reply:
I think it's more productive to grep `source/main` for `choice` (I did that last night). Mind you, I'm not advocating that we limit it just to the new dialog. The opposite, in fact. I'm advocating that if we agree that this is an improvement, that we _make sure_ that all the existing places are affected as well. Consistency for the win!

Also, I just checked who wrote that CSS and it turns out it was me, in #134 :smiley:. So it would've been written specifically for the visibility dialog.

Reply:
I didn't find any additional uses of disabled choices apart from the visibility dialogs you mention (i.e. `setPublic` in `photo.js` and `album.js`).

With 0.6 the `choice` labels have the same color as their description when disabled, while with 0.3 the labels are a shade darker. Obviously I have a slight preference for 0.3 but this is minor and either way is fine. However, I do agree that having the settings consistent across elements is key.